src/test/resources/jsonpatch/testsuite.json

```diff
@@ -1077,6 +1077,180 @@
         } ] ] ]
     },
 
+    {
+        "comment": "test, add, and replace fields",
+        "doc": {
+            "status": "pending",
+            "info": {
+                "name": "John Doe",
+                "age": 30
+            }
+        },
+        "patch": [
+            {"op": "test", "path": "/status", "value": "pending"},
+            {"op": "add", "path": "/info/email", "value": "johndoe@example.com"},
+            {"op": "replace", "path": "/info/age", "value": 31}
+        ],
+        "expected": {
+            "status": "pending",
+            "info": {
+                "name": "John Doe",
+                "age": 31,
+                "email": "johndoe@example.com"
+            }
+        }
+    },
+
+    {
+        "comment": "remove, test, and add nested fields",
+        "doc": {
+            "user": {
+                "credentials": {
+                    "username": "user1",
+                    "password": "pass123"
+                },
+                "profile": {
+                    "email": "user1@example.com",
+                    "phone": "123-456-7890"
+                }
+            }
+        },
+        "patch": [
```

Comment:
Could you please add testcase like:
{"op": "remove", "path": "/user/credentials/password"},
{"op": "add", "path": "/user/credentials/password", "value": "new_passwrod}


I know that we can use `replace` instead of above, but it would be good to have it tested.

Reply:
sure I've added it as a last test

Reply:
it's fine. thanks